src/main/resources/database/changes/release-33/add_category_entry.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO casesvc.category
+VALUES ('FORCED_COMPLETED_TO_NOTSTARTED', 'Forced changed from completed to not started', 'Forced changed to not started', 'ACTIVATED', NULL, NULL, NULL, 'B, BI', NULL, NULL)
```

Comment:
The capitalised status and the description aren't written in great english and should be changed.  Not sure off the top of my head what it should be... if anyone has any ideas before I get some time to think about it then mention it!

Reply:
you might be right! Not great in that :-)